asf_core_data/getters/epc/data_batches.py

```diff
@@ -0,0 +1,139 @@
+# File: asf_core_data/getters/epc/data_batches.py
+"""Look up and integrate batch names."""
+
+# ---------------------------------------------------------------------------------
+
+# Imports
+from asf_core_data import Path
+from asf_core_data.config import base_config
+
+import warnings
+import boto3
+
+# ---------------------------------------------------------------------------------
+
+
+def get_all_batch_names(data_path=None, rel_path=base_config.RAW_DATA_PATH):
```

Comment:
It's a) for consistency (since I"m using data_path and rel_path throghout these scripts) and b) for making it easier to work with both S3 and local data. Ideally, S3 and the local data dir are structured the same way so you can always rely on the default rel_path from the config file. So in practice, you never have to set the rel_path.

It's very convenient if you don't have to specify the relative paths and can just pass 'S3' or the local data dir (containing all data) - and the script will know where to look up the specific files.
However, in order to also allow different dir structures, I give the option of passing the relative path. I don't expect this to be used much, though.

The reasoning behind having a data_path and relative_path stems from the problem of no longer having a PROJECT_DIR "for orientation" since it's now a package. I would have loved to just hard-link all paths in the config file but that doesn't work if you have to rely on relative paths as I already knew that we wouldn't want to keep all these large files in the input dir of every single repository. 

Happy to discuss further and test other options - but while it was a bit tricky to set up, I thought it was quite convenient when actually using the package.